src/redux/slices/mentors/mentorsListSlice.js

```diff
@@ -0,0 +1,38 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const API_URL = 'http://localhost:3000/api/v1';
+
+export const fetchMentorsList = createAsyncThunk(
+  'mentor/fetchMentorsList',
+  async () => {
+    const response = await axios.get(`${API_URL}/mentors`);
+    return response.data;
+  },
+);
+
+const mentorsListSlice = createSlice({
+  name: 'mentorsList',
+  initialState: {
+    mentors: [],
+    status: 'idle',
+    error: null,
+  },
```

Comment:
I noticed you've created a `mentorsListSlice` and changed the redux action type of the `mentorsSlice` to `mentor/fetchMentorsList`. I assume this was done to make it immediately clear that the slice is intended for retrieving the mentor list.